### ISSUE TITLE
Error when importing a cropped image

### ISSUE THREAD

**Issue by zickgraf** [neos/neos-development-collection#2538]
### Description

When exporting and afterwards importing a cropped image, the following error occurs:
```
During the import of the "Sites.xml" from the package "Neos.Demo" an exception occurred:
Error: During import an exception occurred:
"Could not convert target type "Neos\Media\Domain\Model\ImageVariant":
Could not convert target type "Neos\Media\Domain\Model\Adjustment\CropImageAdjustment",
at property path "aspectRatio": No converter found which can be used to convert from
"string" to "Neos\Media\Domain\ValueObject\Configuration\AspectRatio".".,
see log for further information.
```

### Steps to Reproduce

1. Crop an image
2. Export to `Sites.xml`
3. Import from `Sites.xml`

#### Expected behavior

The import finishes successfully.

#### Actual behavior

The error mentioned above occurs.

### Affected Versions

Neos 4.3, probably introduced by this commit: https://github.com/neos/neos-development-collection/commit/f5154fcc6b7f4e645ae59cf5bf88b822a83ec50f#diff-7d59d5cfe0957f575bb69f63a71e06a8

**Comment by arjenhoekema** [comment]
I can confirm this bug/behavior, I created a temporary workaround by adding a dedicated TypeConverter for the AspectRatio which accepts a 'string' as a source type.
Don't know if this is the correct way of fixing this...

**Comment by zickgraf** [comment]
I would be interested in the TypeConverter, could you post it here (or somewhere else like in a pull request or fork)? I tried to write one myself but gave up after some failed attempts.

**Comment by zickgraf** [comment]
Thanks a lot, @arjenhoekema!

**Comment by daniellienert** [comment]
Fixed with #2679
